test/testcafe/framework/page-objects/components/BaseFormObject.js

```diff
@@ -176,10 +176,11 @@ export default class BaseFormObject {
     await this.t.click(this.el.find(CANCEL_BUTTON_SELECTOR));
   }
 
-  getSaveButtonLabel() {
+  getSaveButtonLabel(name = 'Next') {
     // in v3 buttons dont have a value prop
     if (userVariables.v3) {
-      return within(this.el).getByRole('button').textContent;
+      const submitButton = this.getButton(name);
+      return submitButton.textContent;
```

Comment:
Is this change needed? Since there should only be one button on the page and that button should be the "Next" button?

Reply:
I do think in some situations there is more than one button on the page -- is that why you added the query for a specific button by its name @vijetmahabaleshwar-okta?

Reply:
That's right @shawyu-okta. 
On the password enrollment page, the "show/hide" password is a button. And the test fails since there are multiple buttons that match the selector. Had to do this change to get over that issue. @glenfannin-okta